rust/src/treefile.rs

```diff
@@ -481,6 +481,10 @@ impl Treefile {
         )
     }
 
+    pub(crate) fn get_treeref(&self) -> String {
```

Comment:
We could call this `get_ostree_ref()` and let's also try to get in the habit of at least trivial doc comments like `/// Returns the "ref" entry in treefile, or the empty string if unset`.